src/components/ui/upload/PreviewFile.tsx

```diff
@@ -27,13 +28,13 @@ const PreviewFile = ({ file, allowDelete = false, onDelete }: Props) => {
         alt={t('file.preview')}
       />
       {allowDelete && (
-        <button
+        <IconButton
+          className="absolute bottom-0 bg-gray-900 bg-opacity-50 text-white w-full flex justify-center items-center py-2 hover:bg-opacity-80 transition-colors duration-200 delay-100"
           onClick={onDelete}
-          aria-label={t('button.delete_file')}
-          className="absolute bottom-0 right-0 bg-gray-900 bg-opacity-50 text-white w-full flex justify-center items-center py-2 hover:bg-opacity-80 transition-colors duration-200 delay-100"
+          aria-labelledby={t('button.delete_file')}
```

Comment:
```suggestion
          label={t('button.delete_file')}
```

Reply:
Dit heb ik uit deze en in een aparte PR gezet